config/config.yaml

```diff
@@ -1,6 +1,17 @@
-project_code: "sage-bionetworks"
-profile: {{ var.profile | default("default") }}
-region: {{ var.region | default("us-east-1") }}
-template_bucket_name: "bootstrap-awss3cloudformationbucket-114n2ojlbvj21"
-template_key_prefix: {{ environment_variable.TRAVIS_BRANCH | default("testing") }}
-admincentral_cf_bucket: "bootstrap-awss3cloudformationbucket-19qromfd235z9"
+project_code: 'sage-bionetworks'
+profile: {{ var.profile | default('default') }}
+region: {{ var.region | default('us-east-1') }}
+docker_image_tag: {{ environment_variable.DOCKER_IMAGE_TAG | default('staging-latest') }}
```

Comment:
This environment variable gets set in CI/CD and will deploy the API image or update it in the stack.

Reply:
This is the `DOCKER_IMAGE_TAG`. It is the name of the current docker image to use for deploying the api app. With new builds, this changes and we do an update to re-deploy.